cla_public/apps/contact/views.py

```diff
@@ -24,7 +23,7 @@ def add_no_cache_headers(response):
 
 def confirmation_email(data):
     data['case_ref'] = session.checker['case_ref']
-    data['callback_requested'] = data['callback_requested'] == YES
+    data['callback_requested'] = data['contact_type'] in ['callback', 'thirdparty']
     if data['callback_requested']:
         data['safe_to_contact'] = data['callback']['safe_to_contact'] == 'SAFE'
```

Comment:
'safe to contact' needs to look at third party form instead if somebody else is getting the call back